polynote-frontend/polynote/data/data.ts

```diff
@@ -84,6 +84,19 @@ export class IvyRepository extends RepositoryConfig {
         return 0;
     }
 
+    // enable parsing when copy-pasting configurations
+    toJSON(): any {
```

Comment:
Can we narrow the `any` typed here? Something like `{type: "ivy" | "maven" | "pip", resolver: RepositoryConfig}` (just writing from memory, not 100% sure about the syntax)